specification/billing/resource-manager/Microsoft.Billing/preview/2019-10-01-preview/billing.json

```diff
@@ -5395,6 +5534,170 @@
+          "type": "string",
+          "readOnly": true
+        },
+        "enrollmentNumber": {
+          "description": "The enrollment number for which the activity log is being fetched.",
+          "type": "string",
+          "readOnly": true
+        },
+        "friendlyEventName": {
+          "description": "The string describing the activityEventName.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "BillingActivityRelatedProperty": {
+      "description": "The activity related properties.",
+      "type": "object",
+      "properties": {
+        "activityEventName": {
+          "description": "The unique string for this event.",
+          "type": "string",
+          "readOnly": true
+        },
+        "relatedProperties": {
+          "description": "The related specific properties for an event.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/activityEventRelatedProperty"
+          }
+        },
+        "nextLink": {
+          "description": "The link (url) to the next page of results.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "activityEventRelatedProperty": {
+      "description": "The activityEventRelatedProperty key-value pair item.",
+      "type": "object",
+      "properties": {
+        "propertyName": {
+          "description": "Name of the activity event property.",
+          "readOnly": true,
+          "type": "string"
+        },
+        "propertyValue": {
+          "description": "Value of the activity event property.",
+          "readOnly": true,
+          "type": "string"
+        }
+      }
+    },
+    "BillingActivityEventCategoryListResult": {
+      "description": "The list of activity categories.",
+      "properties": {
+        "value": {
+          "description": "The list of activity categories.",
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/BillingActivityEventCategory"
+          }
+        },
+        "nextLink": {
+          "description": "The link (url) to the next page of results.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "BillingActivityEventCategory": {
+      "description": "The properties of an activity event category.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "description": "The properties of an activity event category.",
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/BillingActivityEventCategoryProperties"
+        }
+      }
+    },
+    "BillingActivityEventCategoryProperties": {
+      "description": "The properties of an activity event category.",
+      "properties": {
+        "name": {
```

Comment:
name is a top level property on every resource.  It shouldn't be duplicated inside the property bag.